daprdocs/content/en/operations/monitoring/newrelic.md

```diff
@@ -0,0 +1,347 @@
+---
+type: docs
+title: "How-To: Set-up New Relic to collect and observe metrics, traces and logs from Dapr and the underlying applications"
+linkTitle: "New Relic"
+weight: 2000
+description: "Enable Dapr metrics, events and logs with New Relic Kubernetes integration for Azure Kubernetes Service (AKS) and application traces using OpenTelemetry"
```

Comment:
Does this documentation have to be for AKS? It would be better if we can keep it generic as much as possible (if we have to keep it Kubernetes specific then add that in the title but avoid AKS)

Reply:
agree, not specific to AKS